project2/style.css

```diff
@@ -0,0 +1,66 @@
+.title {
+    font-family: 'ZCOOL QingKe HuangYou', cursive;
+    color: white;
+    font-size: 19pt;
+    line-height: 50px;
+    margin-left: 8px;
+    padding-right: 20px;
+
+}
+
+.rate {
+    font-family: 'Khand', sans-serif;
+    font-size: 15pt;
+    display: table;
+    color: white;
+    position: absolute;
+    left: 150px;
+    top: 100px;
+    bottom: 60px;
+}
+
+.context {
```

Comment:
The word you want to use here (and in the class value in the HTML) is "content" -- as in the content of the card vs the image.

This is just an FYI and has no effect on your grade. 👍🏻